projects/step-core/src/lib/client/generated/models/Bookmark.ts

```diff
@@ -0,0 +1,7 @@
+export type Bookmark = {
+  label?: string;
+  page?: string;
+  link?: string;
+  tenant?: string;
+  icon?: string;
+};
```

Comment:
Everything in the generated folder will be removed once we regenerate the API

Reply:
Moved it to /lib/shared